src/Compilers/Core/Portable/Symbols/Attributes/CommonAttributeData.cs

```diff
@@ -58,10 +58,14 @@ internal virtual bool IsConditionallyOmitted
             get { return false; }
         }
 
-        [MemberNotNullWhen(true, nameof(AttributeClass), nameof(AttributeConstructor))]
+        [MemberNotNullWhen(false, nameof(AttributeClass))]
```

Comment:
`CodeGenerationAttributeData.AttributeConstructor`/`CommonAttributeConstructor` is always null. If you want to keep the contract for `AttributeConstructor` then I'll file an issue on IDE layer.

Reply:
> If you want to keep the contract for AttributeConstructor then I'll file an issue on IDE layer.

Let's do that.